includes/Modules/Analytics.php

```diff
@@ -314,6 +315,11 @@ protected function enqueue_gtag_js() {
 			$gtag_opt = array( 'useAmpClientId' => true );
```

Comment:
Unrelated, but maybe to prevent accidental future issues, would be nice to change this to `$gtag_opt['useAmpClientId'] = true;`, instead of re-initializing the entire array.

Reply:
Yeah, I caught that and would be happy to change 😄